guest-tools/run_td.sh

```diff
@@ -36,7 +36,7 @@ qemu-system-x86_64 -D /tmp/tdx-guest-td.log \
 		   -name ${PROCESS_NAME},process=${PROCESS_NAME},debug-threads=on \
 		   -cpu host \
 		   -object tdx-guest,id=tdx \
-		   -machine q35,kernel_irqchip=split,confidential-guest-support=tdx \
+		   -machine q35,kernel_irqchip=split,confidential-guest-support=tdx,hpet=off \
```

Comment:
could you confirm that in the td_guest.xml.template we already have this option off :
```
  <clock offset='utc'>
    <timer name='hpet' present='no'/>
  </clock
  
```

Reply:
Yes, it's already off in  td_guest.xml.template